src/Orc.NuGetExplorer.Xaml/ViewModels/ExplorerViewModel.cs

```diff
@@ -131,9 +135,26 @@ private void InitializePages()
             _pageSetup.Values.ForEach(page =>
                 Pages.Add(_typeFactory.CreateInstanceWithParametersAndAutoCompletion<ExplorerPage>(page)));
 
+            foreach (var page in Pages)
+            {
+                page.PropertyChanged += OnPageActiveChanged;
+            }
+
             StartPage = _startPage;
         }
 
+        private void OnPageActiveChanged(object sender, PropertyChangedEventArgs e)
+        {
+            if (e.HasPropertyChanged(nameof(ExplorerPage.IsActive)))
+            {
+                var activeTab = Pages.FirstOrDefault(p => p.IsActive)?.Parameters.Tab;
+                if (activeTab is not null)
+                {
+                    ActivatedExplorerTabMessage.SendWith(activeTab);
+                }
```

Comment:
I recommend to move this code into separate method `OnExplorerPageIsActiveChanged()`
it will make code cleaner and reduce nesting

Reply:
Seems reasonable to move, but the naming is too ambigious for me as we have now OnExplorerPagePropertyChanged. 
Name OnExplorerPageIsActiveChanged hits handler convention an sounds like it's work itself on property IsActive change, not called from other handler

Reply:
The aim here is send message, so i think we can call it SendIsActivePageChanged()